Userland/Libraries/LibC/regex.h

```diff
@@ -6,7 +6,7 @@
 
 #pragma once
 
-#include "bits/regex_defs.h"
+#include <LibRegex/RegexDefs.h>
```

Comment:
Will this change affect/reintroduce https://github.com/SerenityOS/serenity/issues/15251?